libs/ui/theme/src/theme.ts

```diff
@@ -12,5 +13,23 @@ export const theme = extendTheme({
         },
       },
     },
+    Button: {
+      baseStyle: {
+        rounded: 'md',
+        size: 'lg',
+        my: 2,
+        fontFamily: 'Abel',
```

Comment:
I know Abel probably picked in Figma but I'm not sure if we actually change the font in our App, we probably use the default React Native Base font